### PR TITLE
adjust disk-size to sum of zeebe and es

### PR DESCRIPTION
**Pull request by hisImminence**
## Description

/closes https://github.com/camunda/doc-hospital/issues/24

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [x] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Review by npepinpe — APPROVED** [review]

**Review by jonathanlukas — APPROVED** [review]

File: docs/components/best-practices/architecture/sizing-your-environment.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ Contact your Customer Success Manager to increase the cluster size beyond the ma
 | Max Throughput **Tasks/second** **\***                                              |                                100 |                                 200 |                              300 |                              400 |
 | Max Throughput **Process Instances/day** **\*\***                                   |                              432 k |                               864 k |                          1.296 k |                          1.728 k |
 | Max Total Number of Process Instances stored (in Elasticsearch in total) **\*\*\*** |                               75 k |                               150 k |                            225 k |                            300 k |
-| Approximate resources provisioned **\*\*\*\***                                      | 11 vCPU, 22 GB memory, 64 GB disk. | 22 vCPU, 44 GB memory, 128 GB disk. | 33 vCPU, 66 GB mem, 192 GB disk. | 44 vCPU, 88 GB mem, 256 GB disk. |
+| Approximate resources provisioned **\*\*\*\***                                      | 11 vCPU, 22 GB memory, 192 GB disk. | 22 vCPU, 384 GB memory, 128 GB disk. | 33 vCPU, 66 GB mem, 576 GB disk. | 44 vCPU, 88 GB mem, 768 GB disk. |
```
Comment (jonathanlukas) [inline docs/components/best-practices/architecture/sizing-your-environment.md:158]: I think the x2 column would not require 384 gb ram, instead the disk space needs to be adjusted:
```suggestion
| Approximate resources provisioned **\*\*\*\***                                      | 11 vCPU, 22 GB memory, 192 GB disk. | 22 vCPU, 44 GB memory, 384 GB disk. | 33 vCPU, 66 GB mem, 576 GB disk. | 44 vCPU, 88 GB mem, 768 GB disk. |
```

File: versioned_docs/version-8.6/components/best-practices/architecture/sizing-your-environment.md [hunk 2]
```diff
@@ -155,7 +155,7 @@ Contact your Customer Success Manager to increase the cluster size beyond the ma
 | Max Throughput **Tasks/second** **\***                                              |                                100 |                                 200 |                              300 |                              400 |
 | Max Throughput **Process Instances/day** **\*\***                                   |                              432 k |                               864 k |                          1.296 k |                          1.728 k |
 | Max Total Number of Process Instances stored (in Elasticsearch in total) **\*\*\*** |                               75 k |                               150 k |                            225 k |                            300 k |
-| Approximate resources provisioned **\*\*\*\***                                      | 11 vCPU, 22 GB memory, 64 GB disk. | 22 vCPU, 44 GB memory, 128 GB disk. | 33 vCPU, 66 GB mem, 192 GB disk. | 44 vCPU, 88 GB mem, 256 GB disk. |
+| Approximate resources provisioned **\*\*\*\***                                      | 11 vCPU, 22 GB memory, 192 GB disk. | 22 vCPU, 384 GB memory, 128 GB disk. | 33 vCPU, 66 GB mem, 576 GB disk. | 44 vCPU, 88 GB mem, 768 GB disk. |
```
Comment (jonathanlukas) [inline versioned_docs/version-8.6/components/best-practices/architecture/sizing-your-environment.md:158]: Same goes for the versioned docs

```suggestion
| Approximate resources provisioned **\*\*\*\***                                      | 11 vCPU, 22 GB memory, 192 GB disk. | 22 vCPU, 44 GB memory, 384 GB disk. | 33 vCPU, 66 GB mem, 576 GB disk. | 44 vCPU, 88 GB mem, 768 GB disk. |
```

**Review by jonathanlukas — CHANGES_REQUESTED** [review]

**Comment by hisImminence** [comment]
Great catch! Thanks @jonathanlukas 🤝

**Review by jonathanlukas — APPROVED** [review]

**Review by mesellings — APPROVED** [review]
Lgtm 👍

**Comment by mesellings** [comment]
Published in the latest docs release today 👍

**Comment by hisImminence** [comment]
Thanks @mesellings!

**Pull request merged**